cmd/utils/nodecmd/chaincmd.go

```diff
@@ -119,7 +119,7 @@ func initGenesis(ctx *cli.Context) error {
 	}
 
 	data := getGovernanceItemsFromGenesis(genesis)
-	gbytes, err := json.Marshal(data)
+	gbytes, err := json.Marshal(data.Items())
```

Comment:
I think that this line has made all PRs reverted. How about add a unit test for this modification?

Reply:
@jeonghonah Actually there are two tests related with this. 
In default_test.go in governance folder, following two tests had added.
These tests will check if the generated genesis hash is same as that of Baobab and Cypress. So if the marshaled string is wrong, its overall hash will also be wrong and won't be able to pass the tests.

```go
func TestBaoBabGenesisHash(t *testing.T) {
	baobabHash := common.HexToHash("0xe33ff05ceec2581ca9496f38a2bf9baad5d4eed629e896ccb33d1dc991bc4b4a")
	genesis := blockchain.DefaultBaobabGenesisBlock()
	genesis.Governance = blockchain.SetGenesisGovernance(genesis)
	blockchain.InitDeriveSha(genesis.Config.DeriveShaImpl)

	db := database.NewMemoryDBManager()
	block, _ := genesis.Commit(db)
	if block.Hash() != baobabHash {
		t.Errorf("Generated hash is not equal to Baobab's hash. Want %v, Have %v", baobabHash.String(), block.Hash().String())
	}
}

func TestCypressGenesisHash(t *testing.T) {
	cypressHash := common.HexToHash("0xc72e5293c3c3ba38ed8ae910f780e4caaa9fb95e79784f7ab74c3c262ea7137e")
	genesis := blockchain.DefaultGenesisBlock()
	genesis.Governance = blockchain.SetGenesisGovernance(genesis)
	blockchain.InitDeriveSha(genesis.Config.DeriveShaImpl)

	db := database.NewMemoryDBManager()
	block, _ := genesis.Commit(db)
	if block.Hash() != cypressHash {
		t.Errorf("Generated hash is not equal to Baobab's hash. Want %v, Have %v", cypressHash.String(), block.Hash().String())
	}
}
```